common/zoom/zoom-user.ts

```diff
@@ -126,22 +113,20 @@ async function getZoomUser(email: string): Promise<ZoomUser> {
     );
 
     if (response.status === 404) {
-        logger.error(`Zoom - No Zoom user found for student with email ${email}`);
+        logger.info(`Zoom - No Zoom user found for student with email ${email}`);
```

Comment:
As we also call this to determine whether a user already exists, this isn't really an error